.mergify.yml

```diff
@@ -69,3 +69,27 @@ pull_request_rules:
         strict: smart
       dismiss_reviews: {}
       delete_head_branch: {}
+  - name: backport patches to release-v2.1 branch
+    conditions:
+      - base=master
+      - label=backport-to-release-v2.1
+    actions:
+      backport:
+        branches:
+          - release-v2.1
+  # automerge backports if CI successfully ran
+  - name: automerge backport release-v2.1
+    conditions:
+      - author=mergify[bot]
```

Comment:
Is the author not @Mergifyio? Github search uses [`author=app/mergify`](https://github.com/ceph/ceph-csi/pulls?q=is%3Apr+author%3Aapp%2Fmergify). I do not think `author=mergify[bot]` is correct, unless it is documented somewhere (link?).

Reply:
This is being used in current code base and things are working fine.https://github.com/ceph/ceph-csi/blob/master/.mergify.yml#L59